.pipelines/dev-tunnels-ssh-ci.yaml

```diff
@@ -75,6 +75,11 @@ jobs:
         inputs:
           verbose: false
       
+      - task: NuGetToolInstaller@1
+        displayName: Use NuGet 6.4.0
+        inputs:
+          versionSpec: '6.4.0'
```

Comment:
`'6.4.x'` is probably better, that way it can pick up patch updates which may include security fixes.

Reply:
Good point - thanks!